app/app.js

```diff
@@ -356,18 +369,25 @@ class MaterialColors {
       $('<div>')
           .addClass(this.CLASS_NAMES.colorTileValueName)
           .text(value.valueName.toUpperCase())
+          .click(() => {
+            let copyText = nunjucks.renderString(
+                this._CONFIG.value_copy_format || "--{{HUE}}-{{VALUE}}",
```

Comment:
Since this copy feature is especially for developers, it'll really helpful to have something close to a possible variable name. I've followed the CSS3 variable naming convention. This can be easily configured easily as I mentioned above.